src/redux/handle-dismiss.js

```diff
@@ -0,0 +1,71 @@
+var keyMirror = require('keymirror');
```

Comment:
I think `handle-dismiss` is a little vague — this reducer might be better named after template-cue (until we have a better name for that, template-cue is a little vague too).